web-common/src/features/dashboards/state-managers/loaders/DashboardStateDataLoader.ts

```diff
@@ -100,84 +96,54 @@ export class DashboardStateDataLoader {
       },
     );
 
-    this.defaultExploreStateAndErrors = getCompoundQuery(
+    this.defaultExploreState = getCompoundQuery(
```

Comment:
Do we need both `this.defaultExploreState` and `this.explorePresetFromYAMLConfig`? Can we remove `this.explorePresetFromYAMLConfig`? If you're keeping this to a separate PR, that's fine 👍.

Reply:
I see, got it